### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

**Comment by distantnative** [comment]
We could also add the first `Description` block as default PR template to https://github.com/getkirby/.github – I think that would be good for any PR in all our repos. The other sections are more specific for this repo.

**Comment by distantnative** [comment]
Do we want to add a step that all/most PRs should be squashed before merging?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ Make sure to point your PR to the relevant develop branches, e.g.
...
 
 
 
-### Why was it changed?
+### Reason
 
 
 
-### What should reviewers know?
+### For reviewers
```
Comment (lukasbestle) [inline .github/PULL_REQUEST_TEMPLATE.md]: What were your thoughts behind making these shorter? TBH I feel they are harder to understand now.

Comment (distantnative) [reply .github/PULL_REQUEST_TEMPLATE.md]: I found it hard to read that first section if all headlines are so similar (What, Why, What...). IMO we should make them easy to parse, even at a glance.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
 A clear and concise description of the PR.
 Use this section for review hints, explanations or discussion points/todos.
 
 Make sure to point your PR to the relevant develop branches, e.g.
 `develop-patch`, `develop-minor` or `v5/develop`
 
+How to contribute: https://contribute.getkirby.com
+-->
+
+### This PR changes…
```
Comment (lukasbestle) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Summary of changes
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,29 +1,45 @@
...
+
+
+
+### Reason
```
Comment (lukasbestle) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Reasoning
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (lukasbestle) [reply .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Additional context
```

**Review by lukasbestle — COMMENTED** [review]
Makes sense. :)

`Suggestions` for a rewording that keeps the distinctive headlines but makes them easier to understand IMO:

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**